modules/aggregations/src/test/java/org/elasticsearch/aggregations/metric/MatrixStatsAggregatorTests.java

```diff
@@ -33,7 +33,7 @@ public void testNoData() throws Exception {
                 indexWriter.addDocument(Collections.singleton(new StringField("another_field", "value", Field.Store.NO)));
             }
             try (IndexReader reader = indexWriter.getReader()) {
-                IndexSearcher searcher = new IndexSearcher(reader);
+                IndexSearcher searcher = newSearcher(reader);
```

Comment:
Looking deeper, I wonder if these aggs tests should rather use a ContextIndexSearcher

Reply:
I think we should do this as part of: #98672
I don't think this should block merging this PR. This is already an improvement.